src/components/Registration/BusinessTypeGP.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div id="business-type">
+    <v-row no-gutters>
+      <v-col cols="12" sm="3" class="pr-4">
+        <label class="d-block title-label">Business Type</label>
+        <v-chip
+          v-if="hasBusinessTypeChecked"
+          id="checked-chip"
+          x-small label
+          color="primary"
+          text-color="white"
+        >
+          Checked
+        </v-chip>
+      </v-col>
+
+      <v-col cols="12" sm="9">
+        <p class="mb-0">General Partnership</p>
+        <div id="business-type-div">
+          <v-checkbox
+            class="business-type-check mt-0"
+            hide-details
+            v-model="checked"
+          />
+          <p class="mb-0" id="business-type-text">
+            I acknowledge that a General Partnership cannot be changed into a
+            Sole Proprietorship (including DBA). If this is necessary, a new
+            Name Request Number and Statement of Registration (along with
+            associated fees) will be required.
+          </p>
+        </div>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue, Watch } from 'vue-property-decorator'
+
+@Component
+export default class BusinessTypeGP extends Vue {
```

Comment:
I just realized that Business Type is an existing property; it is a SP sub-type. (Which explains why you named this BusinessTypeGP.)

I recomment that you rename this component to avoid confusion. What do you plan to call your store value? Perhaps that's a good name for this component too. Maybe something with "acknowledge" or "confirm" in it?

Reply:
I think BusinessConfirm is short and good. I will change it.

Reply:
Yeah, sorry about requesting that change. I see it caused a lot of code changes. But I think the new name is short and good and more clear :)